### PR TITLE
Add a metric that counts false positive partitions

### PR DESCRIPTION
**Pull request by tobim**
### :memo: Checklist

- [ ] All user-facing changes have changelog entries.
- [ ] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [ ] The PR description contains instructions for the reviewer, if necessary.

### CODE REVIEW

File: libvast/src/system/index.cpp [hunk 1]
```diff
@@ -664,6 +668,7 @@ void index_state::send_report() {
   auto query_counters = get_query_counters(pending_queries);
   auto msg = report{
     .data = {
+      {"catalog.lookup.false-positives", counters.partition_false_positives},
```
Comment (dominiklohmann) [inline libvast/src/system/index.cpp:671]: Can you add the query metadata to this metric?

**Review by dominiklohmann — APPROVED** [review]
Fine by me, modulo yet missing documentation. Should we roll this out to the testbed to see how we can plot this?

File: libvast/src/system/index.cpp (hunk 1 above)
Comment (tobim) [reply libvast/src/system/index.cpp:671]: I want to do that. But it requires us to issue it from the query completion function instead. I hope I can get to this on the sooner side.

**Comment by dominiklohmann** [comment]
Closing this as we've identified this approach not to work with the current architecture. The planned catalog centralization will make this easier, and then we'll get back to this.

**Pull request closed without merging**